js/sitnlayers.js

```diff
@@ -1,27 +1,28 @@
+/* global proj4, ol, $ */
 (function (window) {
-  function sitnLayers() {
-    var sitnLayers = {};
-    const _extent = [2420000, 1030000, 2900000, 1350000];
-    const _crs = 'EPSG:2056';
-    const _WMTSurl= 'https://sitn.ne.ch/web_getcapabilities/WMTSGetCapabilities95.xml'
+  function sitnLayers () {
+    var sitnLayers = {}
```

Comment:
Uh ? Are semi-colons not needed anymore ?

Reply:
I'm always forgetting them so I decided to use StandardJS: https://standardjs.com/ linting.
If you want me to use another one, tell me which one and it'll be a pleasure :)